acceptance/logging/logging_test.rb

```diff
@@ -0,0 +1,172 @@
+      sink = logging.create_sink "#{prefix}-sink",
+                                 destination: pubsub_dest,
+                                 filter: "severity = ALERT"
+
+      sink.name.must_equal "#{prefix}-sink"
+      sink.destination.must_equal pubsub_dest
+      sink.filter.must_equal "severity = ALERT"
+
+      sink.filter = "severity >= WARNING"
+
+      sink.save
+
+      sink.name.must_equal "#{prefix}-sink"
+      sink.destination.must_equal pubsub_dest
+      sink.filter.must_equal "severity >= WARNING"
+
+      sink.refresh!
+
+      sink.name.must_equal "#{prefix}-sink"
+      sink.destination.must_equal pubsub_dest
+      sink.filter.must_equal "severity >= WARNING"
+
+      dup_sink = logging.sink "#{prefix}-sink"
+
+      dup_sink.name.must_equal "#{prefix}-sink"
+      dup_sink.destination.must_equal pubsub_dest
+      dup_sink.filter.must_equal "severity >= WARNING"
+
+      logging.sinks.wont_be :empty?
+      logging.sinks(max: 1).length.must_equal 1
+
+      sink.delete
+
+      logging.sink("#{prefix}-sink").must_be :nil?
+    end
+
+    it "lists sinks" do
+      skip
+      pubsub_dest = "pubsub.googleapis.com/projects/#{logging.project}/topics/#{prefix}-topic"
+      sink = logging.create_sink "#{prefix}-list-sink",
+                                 destination: pubsub_dest
+
+      logging.sinks.wont_be :empty?
+      logging.sinks(max: 1).length.must_equal 1
+
+      sink.delete
+    end
+  end
+
+  describe "Metrics" do
+    it "creates, updates, refreshes, gets, lists, and deletes a metric" do
+      metric = logging.create_metric "#{prefix}-metric",
+                                     description: "Metric for acceptance tsets",
+                                     filter: "severity = ALERT"
+
+      metric.name.must_equal "#{prefix}-metric"
+      metric.description.must_equal "Metric for acceptance tsets"
+      metric.filter.must_equal "severity = ALERT"
+
+      metric.description = "Metric for acceptance tests"
+      metric.filter = "severity >= WARNING"
+
+      metric.save
+
+      metric.name.must_equal "#{prefix}-metric"
+      metric.description.must_equal "Metric for acceptance tests"
+      metric.filter.must_equal "severity >= WARNING"
+
+      metric.refresh!
+
+      metric.name.must_equal "#{prefix}-metric"
+      metric.description.must_equal "Metric for acceptance tests"
+      metric.filter.must_equal "severity >= WARNING"
+
+      dup_metric = logging.metric "#{prefix}-metric"
+
+      dup_metric.name.must_equal "#{prefix}-metric"
+      dup_metric.description.must_equal "Metric for acceptance tests"
+      dup_metric.filter.must_equal "severity >= WARNING"
+
+      logging.metrics.wont_be :empty?
+      logging.metrics(max: 1).length.must_equal 1
+
+      metric.delete
+
+      logging.metric("#{prefix}-metric").must_be :nil?
+    end
+  end
+
+  describe "Entries" do
+    let(:proto_payload) { Google::Protobuf::Any.new type_url: "example.com/Greeter/SayHello",
```

Comment:
Would it be better to use a more specific and realistic Protobuf type here? (Note: I'm not sure what that would be, just asking!)